modules/test/playwright/tests/notification-web/emailNotificationTemplate.spec.ts

```diff
@@ -20,6 +20,19 @@ export const test = mergeTests(
 	notificationPagesTest
 );
 
+test.afterEach(async ({notificationTemplatesPage, page}) => {
+	await notificationTemplatesPage.goto();
+
+	const frontEndDatasetItemActions =
+		await notificationTemplatesPage.frontEndDatasetItemAction.all();
+
+	for (let i = 0; i < frontEndDatasetItemActions.length; i++) {
+		await page.getByRole('button', {name: 'Actions'}).first().click();
+
+		await notificationTemplatesPage.frontEndDatasetItemActionDelete.click();
+	}
```

Comment:
Hey @igor-franca, Since we are doing this cleanup through the UI and not the API, I would confirm that these two buttons exist before clicking them, and wrap this entire deletion process through a try catch structure, to easily see errors in case this cleanup fails, what do you think?

```ts
	for (let i = 0; i < frontEndDatasetItemActions.length; i++) {
		try {
			const actionButton = page.getByRole('button', { name: 'Actions' }).first();

			if (actionButton) {
				await actionButton.click();

				const deleteButton = notificationTemplatesPage.frontEndDatasetItemActionDelete;

				if (deleteButton) {
					await deleteButton.click();
				}
			}

		} catch (error) {
			throw new Error(error);
		}
	}
```